### PR TITLE
pkg-mgr: Add digest support and `enable-dependency-version-upgrades` flag

### PR DESCRIPTION
**Pull request by ezgidemirel**

### CODE REVIEW

File: content/v1.16/concepts/packages.md [hunk 1]
```diff
@@ -54,6 +54,20 @@ spec:
   package: xpkg.upbound.io/upbound/platform-ref-aws:v0.6.0
 ```
 
+{{<hint "tip" >}}
```
Comment (jeanduplessis) [inline content/v1.16/concepts/packages.md:57]: Could we add this section for v1.17 as well in `content/v1.17/concepts/packages.md`

File: content/master/software/install.md [hunk 2]
```diff
@@ -259,6 +259,7 @@ at the table below.
 | Alpha | `--enable-realtime-compositions` | Enable support for real time compositions. |
 | Alpha | `--enable-ssa-claims` | Enable support for using server-side apply to sync claims with XRs. |
 | Alpha | `--enable-usages` | Enable support for Usages. |
+| Alpha | `--enable-dependency-version-upgrades ` | Enable automatic version upgrades of dependencies when updating packages. |
```
Comment (jeanduplessis) [inline content/master/software/install.md:262]: Should this flag appear in the 1.17 and 1.16 docs as well?

**Review by jeanduplessis — CHANGES_REQUESTED** [review]

File: content/v1.16/concepts/packages.md (hunk 1 above)
Comment (ezgidemirel) [reply content/v1.16/concepts/packages.md:57]: No, we did not cherry-pick the changes to previous releases

File: content/master/software/install.md (hunk 2 above)
Comment (ezgidemirel) [reply content/master/software/install.md:262]: No, we did not cherry-pick the changes to previous releases

File: content/v1.16/concepts/packages.md (hunk 1 above)
Comment (jeanduplessis) [reply content/v1.16/concepts/packages.md:57]: Should we then maybe not have this addition in the v1.16 version content?

Comment (ezgidemirel) [reply content/v1.16/concepts/packages.md]: thanks for the catch! fixed :)

**Pull request merged**